src/signTxMint.c

```diff
@@ -240,9 +242,26 @@ static void signTxMint_handleToken_ui_runStep()
 		        this_fn
 		);
 	}
+	UI_STEP(HANDLE_TOKEN_STEP_DISPLAY_AMOUNT_WARNING) {
+		const int places = getTokenDecimalPlaces(
+		        &subctx->stateData.tokenGroup,
+		        subctx->stateData.token.assetNameBytes, subctx->stateData.token.assetNameSize
+		);
+		// this warning is only needed when we don't know decimal places
+		if (places != UNKNOWN_DECIMAL_PLACES) {
+			UI_STEP_JUMP(HANDLE_TOKEN_STEP_DISPLAY_AMOUNT);
+		}
+
+		ui_displayPaginatedText(
+			"WARNING: unknown",
+			"decimal places",
+			this_fn
+		);
```

Comment:
I find the warning unavoidable. Because when you get used to common tokens with correct decimal places, and then encounter for the first time a token not in our table, you can rightfully assume that the displayed quantity is also correct, while it might not be (if there are more than 0 decimal places for that token).